cmd/osv-scanner/main.go

```diff
@@ -108,24 +145,58 @@ func getAllCommands(commands []*cli.Command) []string {
 	return allCommands
 }
 
+// Warns the user if a directory with the same name as a subcommand exists.
+func warnIfDirectoryExist(command string, stdout, stderr io.Writer) {
```

Comment:
nit: this is not really just dirs right? but any file? maybe rename to `warnIfCommandAmbigious` to make this even clearer.